files/zh-cn/web/api/viewtransition/finished/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ViewTransition：finished 属性
+short-title: finished
+slug: Web/API/ViewTransition/finished
+---
+
+{{APIRef("View Transitions API")}}{{SeeCompatTable}}
+
+**`finished`** 是 {{domxref("ViewTransition")}} 接口上的一个只读属性，它是一个 {{jsxref("Promise")}}。当过渡动画完成时，该 Promise 就会被兑现，新的页面视图对用户可见且可交互。
+
+仅当 {{domxref("Document.startViewTransition()", "document.startViewTransition()")}} 的回调函数抛出异常或返回一个被拒绝的 Promise 时，`finished` 才会被拒绝，这表示页面的新状态未被创建。
+
+如果过渡动画无法开始，或在动画期间使用 {{domxref("ViewTransition.skipTransition()")}} 跳过了过渡动画，那么视图过渡依旧可以到达最终状态，因此 `finished` 依旧会被兑现。
+
+## 值
+
+一个 {{jsxref("Promise")}}。
```

Comment:
This is a duplicate link (we have referenced this before, let's just follow the en-US document).

```suggestion
一个 promise。
```